### PR TITLE
show map to users if coordinates present

### PR DESCRIPTION
**Pull request by guptapriyanshu7**
Fixes: #1329

https://user-images.githubusercontent.com/60141300/176861049-a623886d-ca8b-427d-b068-92e9f1642f88.mp4

### CODE REVIEW

File: lib/views/after_auth_screens/events/create_event_form.dart [hunk 1]
```diff
@@ -61,8 +61,14 @@ class CreateEventForm extends StatelessWidget {
...
               enabledBorder: InputBorder.none,
               suffix: GestureDetector(
-                onTap: () => navigationService.pushScreen(Routes.mapScreen,
-                    arguments: model),
+                onTap: () => navigationService.pushScreen(
+                  Routes.mapScreen,
+                  arguments: {
+                    'model': model,
+                    'latitude': 37.42796133580664,
+                    'longitude': -122.085749655962,
```
Comment (JamaicanFriedChicken) [inline lib/views/after_auth_screens/events/create_event_form.dart:69]: hey @guptapriyanshu7, i'm going to assume these are placeholder values, can you make a comment above it and just leave a description mentioning these are placeholder values until something else will be in place etc. (something like that)

**Review by JamaicanFriedChicken — CHANGES_REQUESTED** [review]

**Review by JamaicanFriedChicken — APPROVED** [review]

**Comment by TheHazeEffect** [comment]
I'm actually rethinking this implementation. It may be better to just prompt the user to open their maps app (google, waze, apple maps) instead. that way we wouldn't need to add additional configs for google maps or update the CI and we could merge this right away

What do you think @JamaicanFriedChicken ?

**Review by TheHazeEffect — COMMENTED** [review]
Left comment

**Comment by guptapriyanshu7** [comment]
We are using google maps to select a location anyways while creating a new event. We don't need to update the CI btw.

File: lib/views/after_auth_screens/events/create_event_form.dart (hunk 1 above)
Comment (guptapriyanshu7) [reply lib/views/after_auth_screens/events/create_event_form.dart:70]: Added description.

**Comment by palisadoes** [comment]
@TheHazeEffect @JamaicanFriedChicken please review

**Review by JamaicanFriedChicken — APPROVED** [review]

**Review by palisadoes — APPROVED** [review]

**Pull request merged**